homeassistant/components/homematicip_cloud/sensor.py

```diff
@@ -85,7 +90,7 @@
 }
 
 
-async def async_setup_entry(
+async def async_setup_entry(  # noqa: C901
```

Comment:
Not sure if I can fix this easily. Error was:
> homeassistant/components/homematicip_cloud/sensor.py:93:11: C901 `async_setup_entry` is too complex (29 > 25)

Reply:
In a separate PR, I think the function should be refactored. Maybe make a map between classes and entity objects?